chapters/develop-organize/projects/reading/README.md

```diff
@@ -73,8 +95,10 @@ However, for tailored use-cases, they provide a "control panel" in the form of a
 To establish a strong groundwork for frontend checks in a project, using an all-in-one bundle is recommended.
 In our specific context, the foundation for linting relies on [`super-linter`](https://github.com/super-linter/super-linter), and spellchecking is facilitated through [`spellcheck` GitHub action](https://github.com/rojopolis/spellcheck-github-actions), both having minimal setup.
 
+RD: These ^ are typically repository centric, not project specific. I don't think they belong here.
```

Comment:
I added this as a brief example of what an automated validator might look like. To put it simply, a project requires this kind of validation, but it is not mandatory for it to ensure it.